app/controllers/standup_meetings_controller.rb

```diff
@@ -8,12 +8,18 @@ def index
         date: Date.current.strftime('%m-%d-%Y'))
     end
 
-    @standup_meeting_group = StandupMeetingGroup.includes(:users)
```

Comment:
There is no call to `user` on `@standup_meeting_group` throughout its variable life as far as I can tell. Please let me know if I missed something. All existing tests are passing and I’ve verified manually by running the app that the change hasn’t broken anything.